ios/tunnel/doc.go

```diff
@@ -0,0 +1,47 @@
+// Package tunnel implements creating a TUN interface for iOS 17 devices. These tunnels are used to connect to services
```

Comment:
I think the convention is to put this docs into [packagename].go. So tunnel.go in this case

Reply:
both ways work. However, if a package has a longer text (which is of course subjective) it can/should go into the `doc.go` ([godoc](https://go.dev/blog/godoc)). And from my point of view if you open the file and you have to scroll way down to get to the actual code, it's long enough to go into it's own file 🙂